### ISSUE TITLE
Implement WiFi Long Range

### ISSUE THREAD

**Issue by cicciocb** [cicciocb/Annex32_examples#19]
The long range is a feature that increase the range of the wifi but it changes also the format so it is not possible to connect with the module using the standard wifi connection (works only between the modules).
I already tried and found not easy to use / debug.

Do not seems to be a big interest

**Comment by DB9JG** [comment]
Yes, this seems to me a very special use case, which is already covered by other techniques.

**Comment by cicciocb** [comment]
No specific interest, item closed
